src/hover_provider/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { MarkdownString } from "vscode";
+import { NodeMetaType, SourceMetaType } from "../domain";
+
+export function GenerateHoverMarkdownString(
```

Comment:
@mdesmet I am not a javascript developer so keep me honest here. Not sure if the union typing here is a best practice but this is one function doing the markdown for models and source.

@anandgupta42 you want this further refactored to separate the column functionality in a separate function?

Reply:
@jsnb-devoted this looks good for now